EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -219,7 +219,7 @@ QUEST_LV_0300_20151001_000218	Ah, that's it.{nl}Two of them are broken.
 QUEST_LV_0300_20151001_000219	We better ask about the soldiers who created the detectors about them.{nl}I heard the person named Retia is the officer so we may receive some help from her.  
 QUEST_LV_0300_20151001_000220	He is at Verkti Square.{nl}Human matters cannot be resolved by the Kupoles, so we are counting on you.
 QUEST_LV_0300_20151001_000221	Are you talking about the petrification detector at the Coastal Fortress?{nl}We are already aware of that.
-QUEST_LV_0300_20151001_000222	By the way, it hasn't arrived yet? {nl}I've already dispatched the soldiers there.{nl}
+QUEST_LV_0300_20151001_000222	But it hasn't left yet?{nl}I've already dispatched the soldiers there.{nl}
```

Comment:
Try to avoid the use of 'But' in front of the sentence.
Original is good since it says 'arrived'. "Hasn't it arrived yet?"